src/pages/hyunjinKim/Main/Feeds/Reply/Replys.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import Reply from './Reply';
+import ReplyAddForm from './ReplyAddForm';
+
+class Replys extends Component {
+  handleDelete = reply => {
+    this.props.onDelete(reply);
+  };
+
+  handleAdd = replyInput => {
+    this.props.onAdd(replyInput);
+  };
+  render() {
+    return (
+      <>
+        <>
+          {this.props.replys.map(reply => (
+            <Reply
```

Comment:
```suggestion
  render() {
    return (
      <>
          {this.props.replys.map(reply => (
            <Reply
```
fragment 한번만 해도 될거 같아요

Reply:
감사합니다 수정했습니다!ㅎㅎ